packages/react-bindings/src/hooks/client.ts

```diff
@@ -57,9 +57,11 @@ export const useCreateStreamVideoClient = ({
             console.error(`Failed to establish connection`, err);
           });
       }
-      return client.connectUser(user, tokenOrProvider).catch((err) => {
-        console.error(`Failed to establish connection`, err);
-      });
+      if (user.id && tokenOrProvider) {
+        return client.connectUser(user, tokenOrProvider).catch((err) => {
+          console.error(`Failed to establish connection`, err);
+        });
+      }
```

Comment:
this check seems unnecessary. If missing we can throw an error instead. ore clear to the integrator in that way I think

Reply:
I have removed it in my latest commit.